armi/reactor/blueprints/assemblyBlueprint.py

```diff
@@ -174,15 +220,19 @@ def _createBlock(self, cs, blueprint, bDesign, axialIndex):
 
     def _checkParamConsistency(self, a):
```

Comment:
It is somewhat unfair to you, @keckler , this method didn't have unit tests to begin with. But now I'm wondering if we need one to validate your change works.

Reply:
Unit tests are added. Also went ahead and fixed #515 in the process, since it was so related.